src/ripple/app/ledger/Ledger.cpp

```diff
@@ -737,10 +737,10 @@ Ledger::negativeUNL() const
 std::optional<PublicKey>
 Ledger::validatorToDisable() const
 {
-    if (auto sle = readSLE(keylet::negativeUNL());
-        sle && sle->isFieldPresent(sfValidatorToDisable))
+    if (auto nUNLLedgerObj = read<NegUNLKeylet, NegUNLImpl<false>>(keylet::negativeUNL());
```

Comment:
I don't remember the details, but the compiler was throwing an error without an explicit specialization of the `read<>` function template. I'll investigate if I can remove the verbosity.